apps/web/src/app/settings/verify-email.component.html

```diff
@@ -1,21 +1,15 @@
-<div class="card border-warning">
-  <div class="card-header bg-warning text-white">
+<div
+  class="tw-relative tw-flex tw-min-w-0 tw-flex-col tw-break-words tw-rounded tw-border tw-border-solid tw-border-warning-500 tw-bg-background tw-bg-clip-border"
+>
+  <div
+    class="tw-mb-0 tw-bg-warning-500 tw-px-5 tw-py-2.5 tw-font-bold tw-uppercase tw-text-contrast"
+  >
     <i class="bwi bwi-envelope bwi-fw" aria-hidden="true"></i> {{ "verifyEmail" | i18n }}
   </div>
-  <div class="card-body">
+  <div class="tw-p-5">
     <p>{{ "verifyEmailDesc" | i18n }}</p>
-    <button
-      type="button"
-      class="btn btn-block btn-outline-secondary btn-submit"
-      #sendBtn
-      [appApiAction]="actionPromise"
-      [disabled]="$any(sendBtn).loading"
-      (click)="send()"
-    >
-      <i class="bwi bwi-spin bwi-spinner" title="{{ 'loading' | i18n }}" aria-hidden="true"></i>
-      <span>
-        {{ "sendEmail" | i18n }}
-      </span>
+    <button #sendBtn bitButton type="button" [block]="true" [bitAction]="send">
```

Comment:
Do we need a reference to the button?

We can also just use `block` instead of the long hand I think.

```suggestion
    <button bitButton type="button" block [bitAction]="send">
```

Reply:
No reference needed, but there is a lingering Tech Debt tasks that requests all user-interactive elements have an ID to help with automation testing: I've removed the reference and added the `id` attribute. 

I don't see anything in the button component that allows the shorthand `block`. I attempted to use it but was greeted with template errors. Left this one as-is for now.

Reply:
https://github.com/bitwarden/clients/blob/master/libs/components/src/button/button.stories.ts#L94-L95 now that you mention it, the strict templates probably gives you warnings since it expects a bool. We solved it for some other location and I need to go back and update the buttons.

Reply:
Added `block` boolean coercion - I'm hoping this is what you meant by your above comment 👍